src/Aks/Aks.Test/ScenarioTests/AksTestRunner.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using System.Collections.Generic;
+using Microsoft.Azure.Test.HttpRecorder;
+using Microsoft.Azure.Commands.Common.Authentication.Models;
+using System.Reflection;
+using System.IO;
+using Microsoft.Azure.Management.Internal.Resources;
+using Microsoft.Azure.Management.ContainerService;
+using Microsoft.Azure.Management.Authorization.Version2015_07_01;
+using Microsoft.Azure.Commands.Common.MSGraph.Version1_0;
+using Microsoft.Azure.Commands.TestFx;
+using Xunit.Abstractions;
+
+namespace Commands.Aks.Test.ScenarioTests
+{
+    public class AksTestRunner
+    {
+        protected readonly ITestRunner TestRunner;
+
+        protected AksTestRunner(ITestOutputHelper output)
+        {
+            TestRunner = TestManager.CreateInstance (output)
+                .WithNewPsScriptFilename ($"{GetType().Name}.ps1")
+                .WithProjectSubfolderForTests ("ScenarioTests")
+                .WithCommonPsScripts (new[]
+                {
+                    @"Common.ps1",
+                    @"../AzureRM.Resources.ps1"
+                })
+                .WithNewRmModules (helper => new[]
+                {
+                    helper.RMProfileModule,
+                    helper.GetRMModulePath("AzureRM.Aks.psd1")
+                })
+                .WithNewRecordMatcherArguments (
+                    userAgentsToIgnore: new Dictionary<string, string>
+                    {
+                        {"Microsoft.Azure.Management.Resources.ResourceManagementClient", "2017-05-10"},
+                        {"Microsoft.Azure.Management.ResourceManager.ResourceManagementClient", "2017-05-10"}
+                    },
+                    resourceProviders: new Dictionary<string, string>
+                    {
+                        {"Microsoft.Resources", null},
+                        {"Microsoft.Features", null},
+                        {"Microsoft.Authorization", null}
+                    }
+                ).WithMockContextAction(() =>
```

Comment:
Remove the invocation of WithMockContextAction()

Reply:
WithMockContextAction() still not removed

Reply:
I deleted my previous comment by mistake. I mean, I can't pass any ScenarioTests if I remove it, Is there no problem when you run?

Reply:
Looks like ssh related code is necessary.